pkg/pod/pod.go

```diff
@@ -292,15 +353,41 @@ func (builder *Builder) WaitUntilInStatus(status corev1.PodPhase, timeout time.D
 	glog.V(100).Infof("Waiting for the defined period until pod %s in namespace %s has status %v",
 		builder.Definition.Name, builder.Definition.Namespace, status)
 
-	return wait.PollUntilContextTimeout(
+	_, err := builder.WaitUntilInStatuses([]corev1.PodPhase{status}, timeout)
+
+	return err
+}
+
+// WaitUntilInStatuses waits for the duration of the defined timeout or until the pod gets to any specific status
+// in a list of statues.
+func (builder *Builder) WaitUntilInStatuses(statuses []corev1.PodPhase,
```

Comment:
Can you please rename this to WaitUntilInOneOfStatuses ?

Reply:
fixed